core/domain/fs_domain.py

```diff
@@ -165,10 +165,10 @@ def _get_file_metadata(self, filepath, version):
         """
         if version is None:
             return file_models.FileMetadataModel.get_model(
-                self._exploration_id, 'assets/%s' % filepath)
+                self._exploration_id, 'assets/image/%s' % filepath)
```

Comment:
I just realized that this might not be the correct fix. Could you check what happens when you upload an audio file for an exploration locally? Where does it get stored? (Its path should be assets/audio/... but that seems impossible with this change.)

Since the aim is to have a dev file system that has parity with GCS (see GcsFileSystem below) the APIs ought to work the same way.

Reply:
Thanks Sean, you're right. 
I've kept the API unchanged and added a method in fs_services in the latest change.